libnmstate/nm/connection.py

```diff
@@ -206,6 +208,11 @@ def create_new_nm_simple_conn(iface, nm_profile):
             create_macvlan_setting(iface_info, nm_profile, tap=True)
         )
 
+    if Veth.CONFIG_SUBTREE in iface_info:
```

Comment:
how about `if iface.type == InterfaceType.VETH`.

Reply:
Done, I will open a PR modifying this for the other interfaces. I think it is important to keep consistency, thanks!